projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -46,7 +46,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Pipelines.Sockets.Unofficial" Version="2.1.1" />
+    <PackageReference Include="System.Threading.Channels" Version="4.7.0" />
+    <PackageReference Include="Microsoft.IO.RecyclableMemoryStream" Version="1.3.2" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" Condition="$('TargetFramework') == ('net461')" />
```

Comment:
This should not be conditioned.

Reply:
That's part of the "rebase on `master`" work 😄

Reply:
True, but there's no reason to include it for netstandard builds either as it's a noop there anyway. It's simpler to skip the condition though :)

Reply:
FWIW with the condition the build fails in my Linux and OS X environment 🤷‍♂